src/CMakeLists.txt

```diff
@@ -471,6 +471,11 @@ target_compile_options(server PRIVATE "${WARN_MISSING_PROTOTYPES}")
 
 IF (TARGET client)
     TARGET_COMPILE_DEFINITIONS(client PRIVATE USE_CLIENT=1 USE_SAVEGAMES=1 USE_FIXED_LIBAL=1 USE_SDL=1)
+    IF(USE_SYSTEM_OPENAL)
```

Comment:
Would be cleaner if that bit of logic would be moved into `extern/CMakeLists.txt` as well.
Maybe add an alias target to map the "wrong" name or something.

Reply:
@res2k I have addressed the issue by NOT setting `AL_LIBTYPE_STATIC` if `USE_SYSTEM_OPENAL` is `ON`